tests/unit_tests/test_wandb_init.py

```diff
@@ -0,0 +1,48 @@
+import os
+import platform
+import tempfile
+
+import pytest
+import wandb
+
+
+@pytest.mark.skipif(
```

Comment:
can you use the patching solution that Dima suggested, seems better than skipping tests on linux